miso-web/src/main/java/uk/ac/bbsrc/tgac/miso/webapp/controller/EditStudyController.java

```diff
@@ -105,8 +125,35 @@ public Project populateProject(@PathVariable Long projectId) throws IOException
   public Collection<StudyType> populateStudyTypes() throws IOException {
     return studyService.listTypes();
   }
+  
+  /**
+   * Populates 'projects' model attribute with list of projects sorted by name if in plain sample mode, by shortname if in detailed
+   * sample mode.
+   * 
+   * TODO: Would any other parts of MISO benefit from this sorting? Should this be moved where other Controllers can access it?
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1135)

Reply:
Please complete, remove, or make a ticket for this TODO comment